autorest/codegen/models/schema_response.py

```diff
@@ -31,6 +31,15 @@ def __init__(
         self.status_codes = status_codes
         self.headers = headers
         self.binary = binary
+        self.nullable = self.yaml_data.get("nullable", False)
+
+    @property
+    def operation_type_annotation(self) -> str:
```

Comment:
I moved the operation type annotation onto the schema response object itself, because a response can be nullable, but this information won't show up on the schema itself